src/makeInfinityAdder.js

```diff
@@ -4,7 +4,21 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(num) {
+    if (num === undefined) {
```

Comment:
The check for `num === undefined` is correct for determining when to return the accumulated sum. However, consider using `arguments.length === 0` to check if no arguments were passed, which is more explicit and handles cases where `undefined` might be passed intentionally.